### DISCUSSION TITLE
Add Role permissions to Custom Options Page

### DISCUSSION THREAD

**Discussion by millionleaves** [pods-framework/pods#6969] [Ideas]
### Problem to Solve

It would be useful if there was an option to set the minimum role required for someone to make changes to the Custom Options page. Have achieved using User Role Editor. I guess it could be done by choosing either a role or a capability, as seen in regular Pods.

### Proposed Solution

Add a field to let an admin choose the minimum role/capability on the Admin UI tab.

### Possible Workaround

Install **User Role Editor** and assign the capability to the user role(s) that should be able to edit the settings in the custom options page.

**Comment by JoryHogeveen** [comment]
For context:

Pods Setting pages all get their own unique capability (automatically).
It would indeed be nice to have an option to select the available roles that can access the page instead of having to modify the roles manually role editors.
